app/controllers/api/base_controller.rb

```diff
@@ -9,6 +9,7 @@ class BaseController < ApplicationController
     skip_before_action :verify_authenticity_token
     skip_after_action :set_body_class_header
     skip_around_action :mark_notifications_as_read!
+    skip_after_action :updated_last_visited_on!
```

Comment:
I assume we don't want this for API requests? Or do we?

Reply:
I think we want to skip.